<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-11-09T20:41:18 IST" name="Suite" finished-at="2023-11-09T20:41:25 IST" duration-ms="6838">
    <groups>
    </groups>
    <test started-at="2023-11-09T20:41:18 IST" name="Test" finished-at="2023-11-09T20:41:25 IST" duration-ms="6838">
      <class name="testCases.Project_CrudTest">
        <test-method signature="CreateProject(java.lang.String)[pri:0, instance:testCases.Project_CrudTest@1725dc0f]" started-at="2023-11-09T20:41:18 IST" name="CreateProject" data-provider="projectData" finished-at="2023-11-09T20:41:22 IST" duration-ms="3328" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[test987]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateProject -->
        <test-method signature="deleteProject()[pri:0, instance:testCases.Project_CrudTest@1725dc0f]" started-at="2023-11-09T20:41:22 IST" name="deleteProject" finished-at="2023-11-09T20:41:23 IST" depends-on-methods="testCases.Project_CrudTest.CreateProject" duration-ms="1531" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteProject -->
        <test-method signature="getProject()[pri:0, instance:testCases.Project_CrudTest@1725dc0f]" started-at="2023-11-09T20:41:23 IST" name="getProject" finished-at="2023-11-09T20:41:25 IST" depends-on-methods="testCases.Project_CrudTest.CreateProject" duration-ms="1372" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "projectIdResponse.Data.getId()" because the return value of "projectIdResponse.Root.getData()" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "projectIdResponse.Data.getId()" because the return value of "projectIdResponse.Root.getData()" is null
at testCases.Project_CrudTest.getProject(Project_CrudTest.java:57)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getProject -->
        <test-method signature="updateProject(java.lang.String)[pri:0, instance:testCases.Project_CrudTest@1725dc0f]" started-at="2023-11-09T20:41:25 IST" name="updateProject" finished-at="2023-11-09T20:41:25 IST" depends-on-methods="testCases.Project_CrudTest.getProject" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method testCases.Project_CrudTest.updateProject() on instance testCases.Project_CrudTest@1725dc0f depends on not successfully finished methods 
[testCases.Project_CrudTest.getProject() on instance testCases.Project_CrudTest@1725dc0f]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method testCases.Project_CrudTest.updateProject() on instance testCases.Project_CrudTest@1725dc0f depends on not successfully finished methods 
[testCases.Project_CrudTest.getProject() on instance testCases.Project_CrudTest@1725dc0f]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateProject -->
      </class> <!-- testCases.Project_CrudTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
